src/NSrtm.Core/IElevationProvider.cs

```diff
@@ -4,21 +4,11 @@
 namespace NSrtm.Core
 {
     /// <summary>
-    ///     Provides elevation of ground for WGS84 locations.
+    ///     Provides elevation for WGS84 locations.
```

Comment:
Yeah, but we must standardize what coordinates are used - can you suggest better name?